### DISCUSSION TITLE
Any available fan extension cards?

### DISCUSSION THREAD

**Discussion by vimaxus** [Rem0o/FanControl.Releases#1101] [General]
Hello,

I'm trying to find an extension card but all I find on amazon is just knob (or switch) controlled and powered by SATA or the old 4pin12v.
Ideally something like the old [sunbeam theta](url=https://silentpcreview.com/forums/viewtopic.php?t=27679) obviously unavailable.

In the discussions I saw the thermaltake plugin but it's more for RGB and I see lots of compatibility issues on the amazon reviews. Will try it if there is no fan-centered ones available.
The asus fan extension card was a no go. A custom arduino one was mentioned but no replies yet.

Thanks,
Virgil

**Comment by Rem0o** [comment]
Depending on the number of extra fans you need, you might get away with using y-splitters.

**Reply by vimaxus** [reply]
Thank you for the reply.

I'm looking to control them individually (for a furniture) to avoid resonances, at least different speeds. I guess y-splitters would have them in sync. Maybe if I use different models?

**Reply by Rem0o** [reply]
GridV3 support is coming to LHM: https://github.com/LibreHardwareMonitor/LibreHardwareMonitor/pull/812
There is also corsair stuff support with a plugin: https://github.com/iJacks1980/FanControl.CommanderPRO

**Reply by Rem0o** [reply]
Thermaltake one also: https://github.com/fu-raz/FanControlThermaltake
